### DISCUSSION TITLE
Allure to CTRF converter

### DISCUSSION THREAD

**Discussion by pgs352** [ctrf-io/ctrf#22] [Reporters]
Hello, 
Congratulations on the project, showing the test result directly in the PR is an awesome feature. 

Most of my organization uses behave (python) to automate test cases, and we are showing the result thought the Allure formatter. 
Would be awesome to have a converter similar to the junit-ctrf-converter.

**Comment by beyond-danube** [comment]
Hi @Ma11hewThomas 
I have implemented Allure to CTRF converter, which for now supports basic functionality, would be happy if you list it under Community Plugins.

Published to NPM https://www.npmjs.com/package/allure-to-ctrf
GitHub Repo https://github.com/beyond-danube/allure-to-ctrf

It does not support Epic and Story Allure's specific abstractions, while it's matter time and demand to this to add.
I have added some basic tests, so that I'm quite confident in results correctness themsesves.

**Reply by Ma11hewThomas** [reply]
Thank you @beyond-danube! 

This will be listed in community plugins shortly
